modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/resource/v2_0/PriceListResourceImpl.java

```diff
@@ -536,6 +535,9 @@ private CommercePriceList _updateNestedResources(
 						expirationDateConfig.getMinute(),
 						GetterUtil.getBoolean(
 							priceEntry.getNeverExpire(), true),
+						BigDecimal.valueOf(priceEntry.getPrice()),
+						GetterUtil.getBoolean(
```

Comment:
why you don't check id this is the base price list here?

Reply:
Missed, sorry... now I add the check